web/src/lib/components/asset-viewer/asset-viewer.svelte

```diff
@@ -103,13 +105,15 @@
   let shuffleSlideshowUnsubscribe: () => void;
   let previewStackedAsset: AssetResponseDto | undefined;
   let isShowActivity = false;
+  let isShowEditor = false;
   let isLiked: ActivityResponseDto | null = null;
   let numberOfComments: number;
   let fullscreenElement: Element;
   let unsubscribe: () => void;
   let zoomToggle = () => void 0;
   let copyImage: () => Promise<void>;
   $: isFullScreen = fullscreenElement !== null;
+  $: showNavbar = ![isShowEditor].some(Boolean); // can add another variables
```

Comment:
Why is `isShowEditor` put into an array?

Reply:
I suspect that in the future there will be several conditions for hiding the navbar. So I prepared the code to easily add new variables to this condition.